src/utils/dates.js

```diff
@@ -10,3 +10,27 @@ const ONE_DAY_IN_MILLISECONDS = 86400000;
 export function getFutureDate(offset) {
 	return new Date(Date.now() + offset * ONE_DAY_IN_MILLISECONDS);
 }
+
+export function getDaysBetweenDates(startDate, endDate) {
+	// New items are created without a startDate. This avoids null type
+	// error on new item creation.
+	if (!startDate) {
+		return;
+	}
+
+	// the .toDate method converts date formatting to milliseconds.
+	// divinding these days by ONE_DAY_IN_MILLISECONDS, gives you the total
+	// ammount of days after the difference is established.
+	if (startDate && endDate) {
+		const difference = startDate.toDate() - endDate.toDate();
```

Comment:
I would improve the naming for increased readability. Each of these variables should be clear enough even for a non-coder to understand. What is the difference here?